tripal_layout/config/schema/tripal_layout.schema.yml

```
@@ -85,15 +85,51 @@ tripal_layout_field_group_item:
     label:
       type: string
       label: "The label of this field group"
+    label_visibility:
+      type: string
+      label: "Indicates if the label should be visable."
+    empty_label_behavior:
+      type: string
+      label: "Indicates how to handle an empty label"
+    region:
+      type: string
+      label: "Region to put the field group in (i.e. 'Content')"
     weight:
       type: integer
       label: "The weight to give this field group.  Field groups will be displayed in order of their weights with larger weights being lower on the list."
+    classes:
+      type: string
+      label: "space separated list of classes to apply."
     show_empty:
       type: boolean
       label: "Set to true if this field group should be shown even if all of it's children are empty"
     open:
       type: boolean
       label: "Set to true if this field group should be open by default. False if closed"
+    description:
+      type: string
+      label: "Field group caption or text description."
+    desc_visibility:
+      type: string
+      label: "Indicates if the description should be visable."
```

Same typo brought up above by Doug 🙈 
```suggestion
      label: "Indicates if the description should be visible."
```